packages/react-resizable-panels/src/PanelGroup.ts

```diff
@@ -676,7 +676,7 @@ function PanelGroupWithForwardedRef({
         if (prevDeltaRef.current != delta) {
           prevDeltaRef.current = delta;
 
-          if (!layoutChanged) {
+          if (!layoutChanged && delta !== 0) {
```

Comment:
How would this impact the `else` condition below?

Reply:
From what I understand if `delta` is 0 it always means that the mouse is on top of the resize handle, meaning it can't be outside of the resize limits, so constraint violations are set to none.